botfront/cypress/integration/incoming/conversations_navigation.spec.js

```diff
@@ -120,6 +119,7 @@ describe('incoming page conversation tab', function () {
 
     afterEach(function () {
         cy.logout();
+        cy.deleteProject('bf');
```

Comment:
It's better to clean up the state before each test, because we not sure afterEach is called everytime (crash or refresh). So it may cause issues with tests following the one with the failed afterEach

Reply:
fixed